easybuild/tools/github.py

```diff
@@ -958,6 +958,28 @@ def not_eligible(msg):
     return res
 
 
+def list_prs(parameters):
+    """
+    List PRs
+    
+    :param parameters: https://developer.github.com/v3/pulls/#parameters
```

Comment:
`Selection parameters for PRs, see  https://developer.github.com/v3/pulls/#parameters`